gate-keeper/readme.md

```diff
@@ -0,0 +1,103 @@
+## Introduction
+This guide will walk you through the process of building contracts for the gate keeper, deploying them into an existing world running in Docker, and testing their functionality by executing scripts.
+
+The gate keeper allows you to crowdfund items toward a specific goal. For example, if you need a certain amount of resources to build a new ship for the corporation etc.
```

Comment:
can we elaborate this a bit and give some examples here? To say what resources and a story line with respect to game ?

Reply:
Have done :)